doc/instances.md

```diff
@@ -46,120 +46,120 @@ The currently supported keys are:
+`limits.cpu`                                    | string    | -                 | yes           | -                         | Number or range of CPUs to expose to the instance (defaults to 1 CPU for VMs)
+`limits.cpu.allowance`                          | string    | 100%              | yes           | container                 | How much of the CPU can be used. Can be a percentage (e.g. 50%) for a soft limit or hard a chunk of time (25ms/100ms)
+`limits.cpu.priority`                           | integer   | 10 (maximum)      | yes           | container                 | CPU scheduling priority compared to other instances sharing the same CPUs (overcommit) (integer between 0 and 10)
+`limits.disk.priority`                          | integer   | 5 (medium)        | yes           | -                         | When under load, how much priority to give to the instance's I/O requests (integer between 0 and 10)
+`limits.hugepages.64KB`                         | string    | -                 | yes           | container                 | Fixed value in bytes (various suffixes supported, see {ref}`instances-limit-units`) to limit number of 64 KB huge pages (Available huge-page sizes are architecture dependent.)
+`limits.hugepages.1MB`                          | string    | -                 | yes           | container                 | Fixed value in bytes (various suffixes supported, see {ref}`instances-limit-units`) to limit number of 1 MB huge pages (Available huge-page sizes are architecture dependent.)
+`limits.hugepages.2MB`                          | string    | -                 | yes           | container                 | Fixed value in bytes (various suffixes supported, see {ref}`instances-limit-units`) to limit number of 2 MB huge pages (Available huge-page sizes are architecture dependent.)
+`limits.hugepages.1GB`                          | string    | -                 | yes           | container                 | Fixed value in bytes (various suffixes supported, see {ref}`instances-limit-units`) to limit number of 1 GB huge pages (Available huge-page sizes are architecture dependent.)
+`limits.kernel.*`                               | string    | -                 | no            | container                 | This limits kernel resources per instance (e.g. number of open files)
+`limits.memory`                                 | string    | -                 | yes           | -                         | Percentage of the host's memory or fixed value in bytes (various suffixes supported, see {ref}`instances-limit-units`) (defaults to 1GiB for VMs)
+`limits.memory.enforce`                         | string    | `hard`            | yes           | container                 | If `hard`, instance can't exceed its memory limit. If `soft`, the instance can exceed its memory limit when extra host memory is available
```

Comment:
Shouldn't we therefore similarly quote all default values in those tables to keep things consistent?
Effectively quoting the value if it is an actual value and not quoting if it's a description of the default behavior.

Reply:
We could do that, but can we take that as a separate task? This PR is too big to add such major changes without a huge overhead of work. :(

Reply:
Okay, but yeah, please make it a separate task and get that sorted next, I don't like inconsistencies and that's one that will be visible to users.